Sources/Core/iOS/ActivityIndicatorViewObserver.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  ActivityIndicatorViewObserver.swift
+//  Operations
+//
+//  Created by Matthew Holden 16/04/2016.
+//
+//
+
+import UIKit
+
+public protocol ActivityIndicatorViewAnimationInterface {
+    func startAnimating()
+    func stopAnimating()
+}
+
+extension UIActivityIndicatorView: ActivityIndicatorViewAnimationInterface {}
+
+/**
+ An OperationObserverType that can update the state of an associated `UIActivityIndicatorView`
+ when the observed operation starts and finishes.
+ 
+ - note: Any type conforming to `ActivityIndicatorViewAnimationInterface` can be provided as 
+         the activity indicator.
+ */
+public class ActivityIndicatorViewObserver: OperationDidStartObserver, OperationDidFinishObserver {
+
+    private let activityIndicator: ActivityIndicatorViewAnimationInterface
+
+    /**
+     Initialize the observer with an `ActivityIndicatorViewAnimationInterface`-conforming type.
+     - parameter activityIndicator: the object to start and stop animating
+     - returns: an observer
+     - note: The activity indicator's `startAninating` and `stopAnimating` methods 
+             are guaranteed to execute on the main queue.
+     */
+    public init(activityIndicator: ActivityIndicatorViewAnimationInterface) {
```

Comment:
Yeah, that's partly why my comment was with a "?". Naming stuff is hard. Maybe

```swift
let observer = ActivityIndicatorViewObserver(indicator: myIndicator)
```

I guess I am just trying to be less overly verbose and Objective-C-ish.